.github/workflows/docker.yml

```diff
@@ -24,21 +24,27 @@ jobs:
         uses: docker/metadata-action@v3
         with:
           images: ghcr.io/${{ github.repository }}
+          # generate Docker tags based on the following events/attributes
+          tags: |
```

Comment:
is it ok to have multiple tags with the same name & different values?

Reply:
@duke-harlan I followed an example online. After check in another package will be published, I will answer this question after reviewing its tags.